core/src/main/java/org/infinispan/configuration/global/JGroupsConfigurationBuilder.java

```diff
@@ -60,9 +60,8 @@ public StackFileConfigurationBuilder addStackFile(String name) {
       return stackFileConfigurationBuilder;
    }
 
-   public JGroupsConfigurationBuilder transport(Transport transport) {
+   void transport(Transport transport) {
```

Comment:
How about keeping the method publick and change the signature to 
```suggestion
   void transport(JGroupsTransport transport) {
```
so we can use it for testing? 
Also, some users use custom JChannel.

Reply:
Great, I'll look at the PR at the end of the day or tomorrow! Thanks!